cmd/kraft/pkg/pkg.go

```diff
@@ -164,13 +182,46 @@ func (opts *Pkg) Run(cmd *cobra.Command, args []string) error {
 
 	var tree []*processtree.ProcessTreeItem
 
-	parallel := !config.G[config.KraftKit](ctx).NoParallel
+	// parallel := !config.G[config.KraftKit](ctx).NoParallel
```

Comment:
Should this be commented or just removed?

Reply:
Left over artefact from debugging. I'll remove this commenting out.